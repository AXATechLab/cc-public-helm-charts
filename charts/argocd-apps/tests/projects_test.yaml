suite: test argocd-apps applications
templates:
  - templates/projects.yaml
chart:
  version: 1.0.0
release:
  name: my-release
tests:
  - it: should have correct applications
    set:
      projects:
        tool-prod-axa-rev:
          namespace: tool-prod-axa-rev
        productmodeler-prod-green:
          namespace: productmodellingtool-prod-axa-rev
    asserts:
      - matchSnapshot: {}