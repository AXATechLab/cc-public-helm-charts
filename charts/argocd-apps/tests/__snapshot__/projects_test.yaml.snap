should have correct applications:
  1: |
    apiVersion: argoproj.io/v1alpha1
    kind: AppProject
    metadata:
      name: productmodeler-prod-green
      namespace: tool-prod-axa-rev
    spec:
      destinations:
        - name: '*'
          namespace: productmodellingtool-prod-axa-rev
          server: https://kubernetes.default.svc
        - name: '*-applications'
          namespace: tool-prod-axa-rev
          server: https://kubernetes.default.svc
      sourceRepos:
        - '*'
  2: |
    apiVersion: argoproj.io/v1alpha1
    kind: AppProject
    metadata:
      name: tool-prod-axa-rev
      namespace: tool-prod-axa-rev
    spec:
      destinations:
        - name: '*'
          namespace: tool-prod-axa-rev
          server: https://kubernetes.default.svc
        - name: '*-applications'
          namespace: tool-prod-axa-rev
          server: https://kubernetes.default.svc
      sourceRepos:
        - '*'
